### DISCUSSION TITLE
DataView/DataTable should share features?

### DISCUSSION THREAD

**Discussion by justingel** [primefaces/community#1682] [PrimeVue]
I recently used the DataTable in a project and now use the DataView and noticed, that they don't share the same features like `loading`, `filters` etc. 
why is that? shouldn't those components both have the same functionallities? 
the main difference in those both components imo is how the data is displayed.

**Comment by PALBERTEAU** [comment]
That would be great

**Comment by cagataycivici** [comment]
We've considered it but Table has a wider scope, maybe they can be merged in the future by adding list/grid layout to DataTable.

**Reply by PALBERTEAU** [reply]
That's would be awesome if we could have this, like here https://primevue.org/dataview/#layout with the select button that allow to switch layout and keeping the powerful filter system in the datatable.

Thanks for all the work, great library !
